charts/application/templates/_podTemplate.tpl

```diff
@@ -63,7 +63,7 @@ spec:
         {{- end }}
       securityContext:
         {{- if and $i.securityContext $.Values.initDefaults.securityContext }}
-        {{- toYaml (mergeOverwrite $.Values.initDefaults.securityContext $i.securityContext) | nindent 8 }}
+        {{- toYaml (merge $i.securityContext $.Values.initDefaults.securityContext) | nindent 8 }}
```

Comment:
this will be fixed with
- https://github.com/MediaMarktSaturn/helm-charts/pull/163

Is this aligned?
Also in my opinion, it's not a good practice to have too many concerns in one PR.

Reply:
just updated this branch already , PR #163 will be merged before and everything is fine ;)